src/content/docs/more-integrations/open-source-telemetry-integrations/airflow/monitoring-airflow-ot.mdx

```diff
@@ -0,0 +1,142 @@
+import opentelemetryAirflow02 from 'images/opentelemetry_screenshot_airflow_02.webp'
+
+Monitor Airflow with [OpenTelemetry Metrics](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html#setup-opentelemetry) by visualizing tasks, operators, and DAG executions as metrics.
+
+<img
+  title="Screenshot showing sample Airflow DAG runs in New Relic"
+  alt="Screenshot showing sample Airflow DAG runs in New Relic"
+  src={opentelemetryAirflow01}
+/>
+
+## Prerequisites [#prerequisites]
+
+You need to first install the Airflow package with OpenTelemetry using the `otel` extra. This can be done in one of several ways, depending on how you deploy Airflow:
+
+### Installing from PyPi [#install-pypi]
+
+1. Follow the installation instructions from [Airflow's Documentation](https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html).
+2. When installing with pip, add the `otel` extra to the command.
+  a. eg. `pip install "apache-airflow[otel]"`
+
+### Installing from Docker [#install-docker]
+
+1. Setup the [Airflow Docker image](https://airflow.apache.org/docs/docker-stack/index.html) using instructions from [Airflow's Documentation](https://airflow.apache.org/docs/docker-stack/index.html).
+2. Extend the prebuilt image by using a Dockerfile to install the `otel` extra. You can replace the latest tag with your desired version of the Airflow image.
+
+```
+FROM apache/airflow:latest
+RUN pip install --no-cache-dir "apache-airflow[otel]==${AIRFLOW_VERSION}"
+```
+
+## Configuration [#configuration]
+
+You need to configure the Airflow OpenTelemetry metrics to send data to an [OpenTelemetry Collector](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-intro/) that reports data to a New Relic [OTLP Endpoint](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup/#note-endpoints) using a <InlinePopover type="licenseKey" />.
+
+<Callout variant="important">
+  Airflow does not currently supporting sending OpenTelemetry data with authentication headers, making the OpenTelemetry Collector a requirement to authenticate with New Relic.
+</Callout>
+
+### OpenTelemetry Collector Configuration [#configuration-collector]
+
+1. Follow the [basic collector example](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-basic/) to setup your OpenTelemetry Collector.
+2. Configure it with your appropriate OTLP Endpoint. For example, `https://otlp.nr-data.net:4317`.
+3. For authentication, add your <InlinePopover type="licenseKey" /> to the environment variable `NEW_RELIC_LICENSE_KEY` so that it populates the `api-key` header.
+4. Ensure port 4318 on the collector is open and available to the running Airflow instance. (For docker, you may need to use a [docker network](https://docs.docker.com/network/).)
```

Comment:
```suggestion
4. Use port `4318` on the collector to run the Airflow instance. For Docker, you may need to use a [Docker network](https://docs.docker.com/network/).
```

Reply:
I like your formatting better but the content is wrong, flagging this to edit myself.

Reply:
```suggestion
4. Ensure port 4318 on the collector is reachable from the running Airflow instance. (For docker, you may need to use a [docker network](https://docs.docker.com/network/).)
```

Perhaps this? I can't quite seem to word it in a way that sounds clear.

The idea is that the most common issue is networking, where the Airflow container can't reach the collector. The port needs to be open on the collector which requires the config file to be set up right, and for the docker configuration to expose or bind that port. I'm really not sure how best to word this.